package.json

```diff
@@ -174,6 +174,11 @@
         "command": "go.get.package",
         "title": "Go: Get Package",
         "description": "Run `go get -v` on the package on the current line."
+      },
+      {
+        "command": "go.doc",
+        "title": "Go: Doc",
+        "description": "Get go documentation."
       }
```

Comment:
How about `go.get.doc` , `Go: Get Documentation` and `Get documentation using godoc for package`  instead?